emanifest-py/src/emanifest/client.py

```diff
@@ -974,16 +974,35 @@ def get_handler(self, handler_id: str, details: bool = False) -> RcrainfoRespons
         return self.__rcra_request('GET', endpoint)
 
 
-def _parse_url(base_url: str) -> str:
+def _parse_url(base_url: str | None) -> str:
     """emanifest-py internal helper function"""
+    urls = {
+        "PROD": "https://rcrainfo.epa.gov/rcrainfoprod/rest",
+        "PREPROD": "https://rcrainfopreprod.epa.gov/rcrainfo/rest"
+    }
+    if base_url is None:
+        return urls["PREPROD"]
     if "https" not in base_url:
-        urls = {
-            "PROD": "https://rcrainfo.epa.gov/rcrainfoprod/rest",
-            "PREPROD": "https://rcrainfopreprod.epa.gov/rcrainfo/rest"
-        }
         if base_url.upper() in urls:
             return urls[base_url.upper()]
         else:
             return urls["PREPROD"]
     else:
         return base_url
+
+
+def new_client(base_url: str = None, api_id: str = None, api_key: str = None,
```

Comment:
What do you think about adding a global regulator variable for each client, rather than passing that argument in each function? @dpgraham4401

Reply:
@wiljnichepa Sounds like a great idea. my thoughts...

- [ ] it should default to industry.
- [ ] it should still be a boolean like the current regulator flag

I'm working on some other misc fixes. i'll merge/pr

Also can we clean up the commit history? Not sure we need more commits titled "Update README" in our history. 😁

Reply:
Agreed. Industry should be the default, and it should remain a boolean. We'll need to update the README as well since this changes the "getting started" basics. 

No opposition from me re: cleaning up the commit history.